src/modules/print/BasicsPrintView.js

```diff
@@ -267,7 +267,7 @@ class BasicsPrintView extends PrintView {
       this.renderHorizontalBarStat(
         timeInAuto,
         {
-          heading: t('Time In {{automatedLabel}} Ratio', { automatedLabel }),
+          heading: t('Avg. Daily Time In {{automatedLabel}}', { automatedLabel }),
```

Comment:
Do we need a test to assert that this heading remains correct? Just thinking "Time In ..." vs "Avg. Daily Time In" is kinda different so maybe we want to make sure it doesn't accidentally change. Thoughts?